packages/node/src/handlers/call-api.ts

```diff
@@ -1,43 +1,77 @@
 import * as adapter from '@api3/adapter';
-import { OIS, SecurityScheme } from '@api3/ois';
-import { go, retryOnTimeout } from '../utils/promise-utils';
-import { removeKeys } from '../utils/object-utils';
+import { OIS, ReservedParameterName, SecurityScheme } from '@api3/ois';
+import { getResponseParameters, RESERVED_PARAMETERS } from '../adapters/http/parameters';
 import { getEnvValue } from '../config';
+import { API_CALL_TIMEOUT, API_CALL_TOTAL_TIMEOUT } from '../constants';
 import * as logger from '../logger';
-import { getResponseParameters, RESERVED_PARAMETERS } from '../adapters/http/parameters';
 import {
   AggregatedApiCall,
+  ApiCallParameters,
   ApiCallResponse,
+  ChainConfig,
   Config,
   LogsData,
   RequestErrorCode,
   SecuritySchemeEnvironmentConfig,
 } from '../types';
-import { API_CALL_TIMEOUT, API_CALL_TOTAL_TIMEOUT } from '../constants';
+import { removeKeys } from '../utils/object-utils';
+import { go, retryOnTimeout } from '../utils/promise-utils';
 
-function buildOptions(
-  ois: OIS,
-  securitySchemeEnvironmentConfigs: SecuritySchemeEnvironmentConfig[],
-  aggregatedApiCall: AggregatedApiCall
-): adapter.BuildRequestOptions {
-  // Don't submit the reserved parameters to the API
-  const parameters = removeKeys(aggregatedApiCall.parameters || {}, RESERVED_PARAMETERS);
+function addMetadataParameters(aggregatedApiCall: AggregatedApiCall, chain: ChainConfig): ApiCallParameters {
+  const parameters = aggregatedApiCall.parameters;
+  switch (aggregatedApiCall.parameters[ReservedParameterName.RelayMetadata]?.toLocaleLowerCase()) {
```

Comment:
I don't think we should be using locale as it might lead to inconsistent results
```suggestion
  switch (parameters[ReservedParameterName.RelayMetadata]?.toLowerCase()) {
```